api-gateway-routes/Gemfile.lock

```diff
@@ -1,34 +1,41 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    aws-eventstream (1.1.1)
-    aws-partitions (1.443.0)
-    aws-sdk-core (3.113.1)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
-      aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
+    aws-eventstream (1.4.0)
+    aws-partitions (1.1139.0)
+    aws-sdk-core (3.228.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.992.0)
+      aws-sigv4 (~> 1.9)
+      base64
+      bigdecimal
+      jmespath (~> 1, >= 1.6.1)
+      logger
     aws-sdk-lambda (1.39.0)
       aws-sdk-core (~> 3, >= 3.71.0)
       aws-sigv4 (~> 1.1)
     aws-sdk-sqs (1.38.0)
       aws-sdk-core (~> 3, >= 3.112.0)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.2.3)
+    aws-sigv4 (1.12.1)
       aws-eventstream (~> 1, >= 1.0.2)
-    jmespath (1.4.0)
-    minitest (5.15.0)
+    base64 (0.3.0)
+    bigdecimal (3.2.2)
+    jmespath (1.6.2)
+    logger (1.7.0)
+    minitest (5.25.5)
 
 PLATFORMS
   ruby
+  x86_64-linux
```

Comment:
for my understanding: why was this added?

Reply:
oh, I think this got added because I ran `bundle install` on my linux laptop. A mac user would have had their platform get added. Since CI/CD runs on linux, this is fine. I'm updating it to add the macos platform - but I don't believe it causes any issues that wouldn't be automatically resolved the next time someone tries to work on this with a macbook.